ComplyCo/watchman
# Add batch search to Moov webui

Adding batch search to existing Moov page, via a second form.
Added the file-saver package to handle downloading the output CSV.

**Test Plan**
1. Open https://fspu3irmny.us-east-2.awsapprunner.com/ 
2. Use file chooser to select input CSV ([post_input.csv](https://github.com/ComplyCo/watchman/files/13285201/post_input.csv))
3. Verify output file
4. Adjust "name score" and "match threshold", test and verify
   (Previously loaded file is kept until a new file is chosen.)

<img width="1162" alt="Screenshot 2023-11-08 at 2 20 17 PM" src="https://github.com/ComplyCo/watchman/assets/122316787/3186ca22-66d3-4796-985b-b492d90174fc">
